src/main/java/com/sngular/kloadgen/util/JMeterHelper.java

```diff
@@ -4,8 +4,10 @@
 import java.util.Map;
 import java.util.Properties;
```

Comment:
Not from this PR, but this file is missing a license notice

Reply:
done